Abha_Rana/my_profile/main.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+<link href="https://fonts.googleapis.com/css?family=Abril+Fatface|Fredoka+One|Gloria+Hallelujah|M+PLUS+1p|Pacifico|Satisfy|Shadows+Into+Light|Vollkorn&display=swap" rel="stylesheet">
+
+<!-- jQuery library -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+<!-- Latest compiled JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+<body>
+    <nav class="navbar navbar-default" id="nav-bar" style="position: fixed;">
+        <div class="container">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>                        
+            </button>
+            <a class="navbar-brand" href="#"><b>Abha Rana</b></a>
+          </div>
+          <div class="collapse navbar-collapse" id="myNavbar">
+            <ul class="nav navbar-nav navbar-right">
+              <li><a href="#who">WHO</a></li>
+              <li><a href="#what">WHAT</a></li>
+              <li><a href="#where">WHERE</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+        <!-- First Container -->
+        <div class="container-fluid bg-1 text-center" id="who">
+            <h1 class="margin">Who Am I?</h1>
+            <div class="row">
+                <div class="col-sm-4 zoom" style="background-color:palevioletred;">
```

Comment:
get rid of style attribute from html files

Reply:
Still not fixed

Reply:
got rid of inline styling from html files